deploy/ansible/roles/icaro/templates/privacy-gdpr.j2

```diff
@@ -0,0 +1,3 @@
+Privacy policy for {$ICARO_HOSTNAME}
```

Comment:
the correct variable is  {{ icaro.hostname }}

Reply:
done :+1: